katas/content/linear_algebra/index.md

```diff
@@ -320,13 +320,13 @@ Then:
 
 $$\overline{A} =
 \begin{bmatrix}
-    \overline x_{0,0} & \overline x_{0,1} & \dotsb & \overline x_{0,m-1} \\
-    \overline x_{1,0} & \overline x_{1,1} & \dotsb & \overline x_{1,m-1} \\
+    \overline{x}_{0,0} & \overline{x}_{0,1} & \dotsb & \overline{x}_{0,m-1} \\
+    \overline{x}_{1,0} & \overline{x}_{1,1} & \dotsb & \overline{x}_{1,m-1} \\
     \vdots & \vdots & \ddots & \vdots \\
-    \overline x_{n-1,0} & \overline x_{n-1,1} & \dotsb & \overline x_{n-1,m-1}
+    \overline{x}_{n-1,0} & \overline{x}_{n-1,1} & \dotsb & \overline{x}_{n-1,m-1}
```

Comment:
This and above is _reverting_ a change I made in the prior PR where I had to remove the punctuation to stop Markdown seeing the `_` chars as emphasis delimiters. No longer needed!